src/main/java/org/powerflows/dmn/io/yaml/YamlDecsionConverter.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2018-present PowerFlows.org - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.powerflows.dmn.io.yaml;
+
+import org.powerflows.dmn.engine.model.decision.Decision;
+import org.powerflows.dmn.io.DecisionToExternalModelConverter;
+import org.powerflows.dmn.io.yaml.model.YamlDecision;
+
+public class YamlDecsionConverter implements DecisionToExternalModelConverter<YamlDecision> {
+    @Override
+    public YamlDecision to(final Decision decision) {
+        return null;
```

Comment:
I suggest for next time throws UnsupportedOperationException, even if this is temporary.

Reply:
ok